server/routes/users.ts

```diff
@@ -28,4 +28,28 @@ export async function userRoutes(app: FastifyInstance) {
       return response.users[id];
     }
   );
+
+
+  app.post<{ Body: { id: string } }>(
+    "/login",
+    async (req, res) => {
+      const { id } = req.body;
+      if (id == null || id === "") {
+        return res
+          .status(400)
+          .send({ errorMessage: `Please pass valid user signup parameters!` });
+      }
+
+      const {users: [user]} = await streamChat.queryUsers({id})
```

Comment:
something for future optimization, once the first version of our application is created, we will add our own logic and enhancements

https://www.notion.so/BE-Password-Based-Authentication-feature-6cc5095dc33f40e38eb9d316c8d91d70?pvs=4